capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './nameable'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] According to the requirements, only the first letter of the name should be capitalized. Using ```upcase``` would capitalize all the letters. Kindly use ```capitalize``` instead of ```upcase```

Reply:
fixed